taskvine/src/manager/taskvine.h

```diff
@@ -48,6 +48,11 @@ typedef enum {
 	VINE_SUCCESS_ONLY = 8, /**< Only return this output file if the task succeeded. */
 } vine_file_flags_t;
 
+typedef enum {
+	VINE_SHAREOK = 0, /**< Schedule this file to be shared between peers where available. (default) **/
```

Comment:
VINE_PEER_SHARE, VINE_PEER_NOSHARE   so that it look similar to VINE_CACHE, VINE_NOCACHE?